src/main/kotlin/com/watermelon/context/toolWindow/MyToolWindowFactory.kt

```diff
@@ -39,41 +46,201 @@ class MyToolWindowFactory : ToolWindowFactory {
     override fun shouldBeAvailable(project: Project) = true
 
     class MyToolWindow(toolWindow: ToolWindow) {
+        val backendUrl = "https://app.watermelontools.com"
```

Comment:
I tested this both with the prod and the localhost backendUrl. In both cases I get this 404 page. 

Uploading Screen Recording 2023-09-14 at 10.50.11 AM.mov…

Reply:
I see it even in prod
https://app.watermelontools.com/intellij

Reply:
Can't reproduce.